contracts/ColonyDataTypes.sol

```diff
@@ -172,12 +175,19 @@ contract ColonyDataTypes {
     uint256 blockTimestamp;
   }
 
+  struct Payment {
+    address recipient;
+    bool finalized;
```

Comment:
If we'll only have the 1tx flow then this doesn't apply. But otherwise if we have intermediary state Payment and want to "cancel" it the only reason we'd ever want to do that is to recover tokens from its funding pot. However for Payments, we can move funds around any way we want including emptying its fundings which defies the point of including extra logic around "cancelling" to reclaim.